build/Rulesets/Roslyn.ruleset

```diff
@@ -126,8 +126,8 @@
     <Rule Id="IDE0019WithoutSuggestion" Action="Error" />
     <Rule Id="IDE0020" Action="Error" />                      <!-- Use pattern matching                         if (control is Button)                              if (control is Button button)         -->
     <Rule Id="IDE0020WithoutSuggestion" Action="Error" />
-    <Rule Id="IDE0028" Action="Error" />                      <!-- Use coalesce expression                      string y = x == null ? string.Empty : x;            string y = x ?? string.Empty;         -->
-    <Rule Id="IDE0028WithoutSuggestion" Action="Error" />      
+    <Rule Id="IDE0028" Action="Info" />                       <!-- Collection initialization can be simplified                                                                                            -->
```

Comment:
Oh I see - since you are running integration tests against latest roslyn, this will fail otherwise? Let\s bump to IDE0029 then